person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_accessor :id, :name, :age
+
+  def initialize(id, name, age, parents_permission = 'Unknown')
+    @id = id,
+          @name = name,
+          @age = age,
+          @parents_permission = parents_permission
+  end
+
+  private
+
+  def of_age?
+    if @age >= 18
+      puts true
+    else
+      puts false
+    end
```

Comment:
- Good job on the work done so far, but to help make your code more readable I suggest that the `of_age?` method returns if the age >=18 instead of printing it. PLease fix this 🙏